string[] disk_names
geometry_msgs/Point[] disk_points